File: js/programmers/level.1/prgm_77484_1.js
function solution(lottos, win_nums) {
  const correct = lottos.filter((num) => win_nums.includes(num)).length;
  const zeros = lottos.filter((num) => num === 0).length;

  let min = 7 - correct >= 6 ? 6 : 7 - correct;
  let max = min - zeros < 1 ? 1 : min - zeros;

  return [max, min];
}

File: python/programmers/level.1/prgm_77484_1.py
def solution(lottos, win_nums):
    correct = len([num for num in lottos if num  in win_nums])
    zeros = lottos.count(0)

    min = 6 if 7 - correct >= 6 else 7 - correct
    max = 1 if min - zeros < 1 else min - zeros

    return [max, min]